docs/sources/helm-charts/mimir-distributed/get-started-helm-charts/_index.md

```diff
@@ -301,54 +259,25 @@ Make a choice based on whether or not you already have a Prometheus server set u
 
      > **Note:** On Linux systems, if \<ingress-host\> cannot be resolved by the Prometheus server, use the additional command-line flag `--add-host=<ingress-host>:<kubernetes-cluster-external-address>` to set it up.
 
-### Configure Grafana Agent to write to Grafana Mimir
-
-You can either configure Grafana Agent to write to Grafana Mimir or [configure Prometheus to write to Mimir](#configure-prometheus-to-write-to-grafana-mimir). Although you can configure both, you do not need to.
-
-Make a choice based on whether or not you already have a Grafana Agent set up:
-
-- For an existing Grafana Agent:
+### Configure Grafana Alloy to write to Grafana Mimir
 
-  1. Add the following YAML snippet to your Grafana Agent metrics configurations (`metrics.configs`):
-
-     ```yaml
-     remote_write:
-       - url: http://<ingress-host>/api/v1/push
-     ```
+You can either configure Grafana Alloy to write to Grafana Mimir or [configure Prometheus to write to Mimir](#configure-prometheus-to-write-to-grafana-mimir). Although you can configure both, you don't need to.
 
-     In this case, your Grafana Agent will write metrics to Grafana Mimir, based on what is defined in the existing `metrics.configs.scrape_configs` configuration.
+To configure Grafana Alloy to write to Mimir, use the `prometheus.remote_write` component. For example:
```

Comment:
Let's verify that this is the correct confirguration.

Reply:
Seems to be pulled directly from https://grafana.com/docs/alloy/latest/reference/components/prometheus/prometheus.remote_write/ so seems right.